guides/common/modules/proc_configuring-kerberos-authentication-for-remote-execution.adoc

```diff
@@ -15,11 +15,18 @@ You can use Kerberos authentication to establish an SSH connection for remote ex
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
 # {installer-scenario} \
- --foreman-proxy-plugin-remote-execution-script-ssh-kerberos-auth true
+--foreman-proxy-plugin-remote-execution-script-ssh-kerberos-auth true
```

Comment:
Also related to the above comment, even though it is not changed in this PR. We can, of course, have further discussion.